File: 4st/Jisoo/programmer/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        for(int i=0;i < my_string.length();i++){
            char c=my_string.charAt(i);
            if(c >='a' && c<='z'){
                c-=32;
            }
            else{
                c+=32;
            }
        
            answer=answer+c;
        }
        return answer;
    }
}

File: 4st/Hani/programmer/대문자와 소문자.py
def solution(my_string):
    answer = ''
    
    for i in range(len(my_string)) :
        if(my_string[i].isupper()) :
            answer += my_string[i].lower()
        elif(my_string[i].islower()) :
            answer += my_string[i].upper()
    return answer